jx_sqlite/aggs_table.py

```diff
@@ -205,9 +205,15 @@ def _get(row):
             elif len(edge_names) > 1:
                 domain_names = ["d" + text_type(edge_index) + "c" + text_type(i) for i, _ in enumerate(edge_names)]
                 query_edge.allowNulls = False
+                domain_columns = [c for c in self.sf.columns if quote_table(c.es_column) in vals]
+                if domain_columns and path==domain_columns[0].nested_path[0]:
+                    domain_table = quote_table(frum)
+                else:
+                    domain_table = quote_table(self.sf.fact)
+
```

Comment:
OK, I better understand now; you are hunting down the table that has column which represents the domain.  Can you do the same without a `if/else` block?

                domain_columns = [c for c in self.sf.columns if quote_table(c.es_column) in vals]
                domain_nested_path = domain_columns[0].nested_path
                domain_table = quote_table(concat_field(self.sf.fact, domain_nested_path[0]))

This form may help point out a potential other problem:  `if not domain_columns:  Log.error("expecting a known column")`